styles/helpers/ugc.css

```diff
@@ -0,0 +1,40 @@
+.ugc h1,
```

Comment:
Didn't seem to be the norm in the rest of the files so I followed the convention.